src/components/Search/index.tsx

```diff
@@ -168,6 +169,24 @@ function Search({queryJSON, searchResults, onSearchListScroll, contentContainerS
     const [transactions] = useOnyx(ONYXKEYS.COLLECTION.TRANSACTION, {canBeMissing: true});
     const previousTransactions = usePrevious(transactions);
     const [reportActions] = useOnyx(ONYXKEYS.COLLECTION.REPORT_ACTIONS, {canBeMissing: true});
+
+    const [archivedReportsIDList = getEmptyArray<string>()] = useOnyx(ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS, {
+        canBeMissing: true,
+        selector: (all): ArchivedReportsIDList => {
+            if (!all) {
+                return [];
+            }
+
+            const result: ArchivedReportsIDList = [];
+            for (const [key, value] of Object.entries(all)) {
+                if (value?.private_isArchived) {
```

Comment:
Maybe it's better to use `ReportUtils.isArchivedReport()` for this? The only reason I say that is because if someday, the way we check for archived reports changes, it would be easier to refactor.